examples/DS3231_Temperature/DS3231_Temperature.ino

```diff
@@ -0,0 +1,164 @@
+// CONNECTIONS:
+// DS3231 SDA --> SDA
+// DS3231 SCL --> SCL
+// DS3231 VCC --> 3.3v or 5v
+// DS3231 GND --> GND
+
+/* for software wire use below
+#include <SoftwareWire.h>  // must be included here so that Arduino library object file references work
+#include <RtcDS3231.h>
+
+SoftwareWire myWire(SDA, SCL);
+RtcDS3231<SoftwareWire> Rtc(myWire);
+ for software wire use above */
+
+/* for normal hardware wire use below */
+#include <Wire.h> // must be included here so that Arduino library object file references work
+#include <RtcDS3231.h>
+RtcDS3231<TwoWire> Rtc(Wire);
+/* for normal hardware wire use above */
+
+
+void setup () 
+{
+    Serial.begin(57600);
+    while (!Serial) delay(250);    // Wait until Arduino Serial Monitor opens
+    Serial.println( " " );         // For ESP8266, newline after bootup message
```

Comment:
don't include platform specific comments/code in examples as it makes them harder to read.
remove the comment at least; also, `Serial.println()` is good enough, no need to print the space.

Reply:
Will remove '#if defined(ESP8266)' / '#endif'
Regarding Serial.println( " " )...
I'm using the both the Huzzah breakout and feather boards from Adafruit, which use the ESP12 part. Upon bootup, the ESP12 sends a long message at 74880 baud, which generates overrun/framing errors for the connected serial port when its baud-rate doesn't match. As a result, the newline character sent by a 'Serial.println()' is occasionally  missed, 
Inserting the space clears these errors, insuring the subsequent newline character is seen.

I've left the Serial.println( " " ) intact, but have removed the comment.